frontend/components/OrgPage/ChildDisplay.js

```diff
@@ -0,0 +1,124 @@
+import React, { useState, Component } from 'react'
+import OrgChildren from './OrgChildren'
+import { Link } from '../../routes'
+import {
+  LIGHT_GRAY,
+  BLACK,
+  CLUBS_GREY_LIGHT,
+  WHITE,
+  BABY_BLUE,
+} from '../../constants/colors'
+import s from 'styled-components'
+import { Icon } from '../common'
+const Child = s.div`
+  display: flex;
+  flex-direction: row; 
+  justify-content: space-between;
+  margin: 5px;
+  margin-left: 30px;
+  margin-right: 0px; 
+  border-radius: 10px;
+  background: ${WHITE};
+  padding: 20px; 
+  border: 1px ${BLACK} solid
+
+  &:hover{
+    background: ${LIGHT_GRAY}
+    color: ${BLACK}
+  }
+`
+const Text = s.p`
+  color: ${BLACK};
+  font-size: 16px;
+
+  &:hover{
+  }
+`
+const TextLink = s.a`
+  color: ${BLACK};
+  font-size: 16px;
+
+  &:hover{
+    font-weight: bold;
+    
+  }
```

Comment:
Think this is kinda an odd UX. Would prefer a change in color. Also if the default link is black I'm not sure it's intuitive to the user that this is actually a link